src/screens/signup/SignupHeader.tsx

```
@@ -0,0 +1,21 @@
+  onPrev: () => void,
+}
+
+export default function SignupHeader({ step, onPrev }: SignupHeaderProps) {
+  const nowStep = Math.floor(step);
+  
+  return(
+    <View style={{ width: '100%', flexDirection: "row", alignItems: "center", justifyContent: "space-between", height: 40, marginBottom: 16 }}>
+      <Pressable onPress={onPrev}>
+        <Image source={btn_previous_default} style={{ width: 10, height: 17}} />
+      </Pressable>
+      {step !== 4 && <Typography children={`${nowStep}/3`} font='medium16' color='defaultBlack' />}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)